app/src/main/java/science/apolline/view/activity/MainActivity.kt

```diff
@@ -98,18 +91,18 @@ class MainActivity : RootActivity(), NavigationView.OnNavigationItemSelectedList
                 invalidateOptionsMenu()
             } else if (BluetoothLeService.ACTION_GATT_DISCONNECTED.equals(action)) {
                 mConnected = false
-                if(destroyed == false) {
+                if (destroyed == false) {
                     val builder = AlertDialog.Builder(this@MainActivity)
                     builder.setTitle(R.string.disconnected_sensor_display_title)
                     builder.setMessage(R.string.disconnected_sensor_display_message)
-                    builder.setPositiveButton("reconnecter"){dialog, which ->
+                    builder.setPositiveButton("reconnecter") { dialog, which ->
                         registerReceiver(this, MainActivity.makeGattUpdateIntentFilter())
                         if (MainActivity.mBluetoothLeService != null) {
-                            val result = MainActivity.mBluetoothLeService!!.connect(mPrefs.getString("sensor_mac_address","address not found"))
+                            val result = MainActivity.mBluetoothLeService!!.connect(mPrefs.getString("sensor_mac_address", "address not found"))
                             Log.d(MainActivity.TAG, "Connect request result=$result")
                         }
                     }
-                    builder.setNegativeButton("Ok"){dialog,which ->
+                    builder.setNegativeButton("Ok") { dialog, which ->
```

Comment:
Avoid strings in code, use the [Strings.xml](https://github.com/Apolline-Lille/apolline-android/blob/master/app/src/main/res/values/strings.xml) file instead